modules/test/playwright/tests/frontend-taglib-clay/managementToolbar.spec.ts

```diff
@@ -54,18 +51,140 @@ test.describe('Management Toolbar With Results', () => {
 				.getByRole('tablist')
 				.getByText('Management Toolbars');
 
-			await expect(tabHeading).toBeInViewport();
+			await expect(tabHeading).toBeVisible();
 
 			await tabHeading.click();
 		});
+	}
+);
+
+test.describe('Management Toolbar Default State', () => {
+	test('Assert the New button is displayed as a text button @LPS-144540', async ({
```

Comment:
We can add tags more explicitly, as an additional argument. [See here for example](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts#L205). That way they highlight nicely in PW browser.

Reply:
Highlight actually still works with using `@` in the description, but it does clean the description up so the tags aren't repeated like so:

<img width="1000" alt="Screenshot 2024-10-17 at 2 54 13 PM" src="https://github.com/user-attachments/assets/1d565cbc-9f57-4d2a-a42f-e63cf2b93f39">